doc/supported_file_formats.adoc

```diff
@@ -0,0 +1,144 @@
+| Description
+| Safe?
+
+| -strip all
+| Strips all image metadata your image editor has embedded into the image.
+| Yes (Image metadata is mostly useless for images used with Minetest)
+
+| -o7 -zm1-9
+| Sets both optipng and zlib's optimization/compression levels to the highest available. Will take significantly longer but can further reduce the file size.
+| Yes
+
+| -nc
+| Disables color type reduction. This is necessary if you have color data you want to preserve in transparent pixels, such as ones in leaf textures.
+| N/A
+|===
+
+A general-purpose script for optimizing images as aggressively as possible without losing color information can be found in Minetest Game: https://github.com/minetest/minetest_game/blob/master/utils/optimize_textures.sh[optimize_textures.sh]
+
+==== TGA
+Minetest supports Type 10 TGA files which use RLE (Run-Length-Encoding) for compression. This will lead to a significantly smaller filesize when generating images with large areas of a same color, and is recommended when writing images that do not need to be read back again programmatically.
+
+=== Third-party mod libraries
+If you would like to write or read image files in Lua for a format the engine does not support, there are various libraries available as mods you can use.
+
+==== PNG
+===== Writing & Reading
+* `modlib` (https://content.minetest.net/packages/LMD/modlib/[ContentDB], https://github.com/appgurueu/modlib[GitHub])
+
+==== TGA
+===== Writing
+* `tga_encoder` (https://content.minetest.net/packages/erlehmann/tga_encoder/[ContentDB] https://git.minetest.land/erlehmann/tga_encoder[MeseHub])
+** Supports writing all TGA formats Minetest supports.
+
+== Models
+Minetest's support of various model formats can be seen as lacking. This is a side-effect of Minetest relying on the aging Irrlicht graphics engine which mostly only has support for old or obsolete model formats.
+
+[grid="all"]
+[options="header",stripes=even,cols="^.^,<.^,^.^,^.^,^.^,^.^,^.^,^.^"]
+|===
+| Format
+^| General Usecase
+| File Format
+| Are static meshes supported?
+| Are animated meshes supported?
+| Implementation limitations
+| Blender importer
+| Blender exporter
+
+| OBJ (`.obj`)
+| Static meshes
+| Text
+| Yes
+| No
+| `.mtl` material files are not supported.
+2+| Built-in
+
+| Blitz3D model (`.b3d`)
+| Animated meshes
+| Binary
+| Yes
+| Yes
+| N/A
+.2+| None
+| https://github.com/GreenXenith/io_scene_b3d[io_scene_b3d]
+
+| DirectX 2.0 model (`.x`)
+| Animated meshes in text format
+| Binary/Text
+| Yes
+| Yes
+| N/A
+| https://github.com/minetest/io_scene_x[io_scene_x] (with caveats; see README.md)
+|===
+
+=== Third-party mod libraries
+==== Blitz3D model
+===== Writing & Reading
+* `modlib` (https://content.minetest.net/packages/LMD/modlib/[ContentDB], https://github.com/appgurueu/modlib[GitHub])
+
+== Audio
+Currently the only audio format Minetest supports is OGG Vorbis (`.ogg`). If the sound is to be used as a positional sound it must be mono.
+
+== Schematics
```

Comment:
Considering that schematics are mentioned here, why not mention locale files too?